lib/shopify-cli/messages/messages.rb

```diff
@@ -42,6 +42,8 @@ module Messages
           project_type_select: "What type of project would you like to create?",
         },
 
+        development_version_warning: "{{yellow:You are running the development version of the CLI}}",
```

Comment:
Suggestion: use the star symbol instead of yellow text.

```suggestion
        development_version_warning: "{{*}} You are running the development version of the CLI",
```

Reply:
Is there a concern that they might miss the message if it scrolls past the end of the terminal window and it's just highlighted by a star?  If the colour is different, it'll catch my eye and make me scroll back to see it.

Reply:
The star is yellow and pretty eye-catching (I think?) — I'm mainly thinking about trying to be consistent with how we format particular kinds of text and how we use colour. (I am working on this section of the UX contributor guide 🙂.)

Reply:
Does the star make the warning noticeable enough? It seems to me like it might get drowned out when we call more verbose commands.

Take for instance this `system` call, to me it doesn't stand out enough on a cursory look. I'm happy to keep it this way if people disagree, though 🙂 .

<img width="529" alt="Screen Shot 2020-06-09 at 4 22 02 PM" src="https://user-images.githubusercontent.com/64600052/84195913-5b693480-aa6d-11ea-8a77-c352e5eaa12e.png">

Reply:
That's fair. I didn't realize this would be used with lots of other text following. I might suggest adding an extra line break after the line, but I'm fine with using yellow text. It does seem like that would work better here.

Reply:
I just tried it with the `{{*}}` and it really doesn't 'pop' for me in terms of being a warning.

I'm inclined to make it a fairly big banner so that it's very obvious that the end-user is running a development version:

```
        development_version_warning: <<~DEVELOPMENT,
          {{*}} {{yellow:You are running a development version of the CLI:}}
          {{*}}   {{yellow:%s}}

        DEVELOPMENT
```
And pass in the following as the argument (i.e., the path to the CLI that's currently being run)
```
File.join(ShopifyCli::ROOT, 'bin/shopify-cli')
```